### PR TITLE
Update PR template for dart fix

### PR DESCRIPTION
**Pull request by Piinks**
I don't know if this is feasible, but figured it was worth seeing.
Often times dart fixes are forgotten as part of the code review process.
I know some people ignore the PR template, but considered maybe a mention was warranted.

I've made the change reflected below:

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] I've accounted for any new deprecations or breaking changes (Choose one).
  - [ ] I have added a dart fix, see [Data Driven Fixes].
  - [x] Dart fix cannot help (should this change be reconsidered?), or this change does not need one.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[Data Driven Fixes]: https://github.com/flutter/flutter/wiki/Data-driven-Fixes

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -13,6 +13,9 @@
 - [ ] I listed at least one issue that this PR fixes in the description above.
 - [ ] I updated/added relevant documentation (doc comments with `///`).
 - [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
+- [ ] I've accounted for any new deprecations or breaking changes (Choose one).
+  - [ ] I have added a dart fix, see [Data Driven Fixes].
+  - [ ] Dart fix cannot help (should this change be reconsidered?), or this change does not need one.
```
Comment (goderbauer) [inline .github/PULL_REQUEST_TEMPLATE.md:18]: How about condensing this down to one check box:

```suggestion
- [ ] I followed the [breaking change policy] and added [Data Driven Fixes] where necessary.
```

I am not in love with the word "necessary" there. Maybe "appropriate" or "required" or "possible" would be better?

Comment (Piinks) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: Oo love that! How about `necessary` --> `supported`?

Comment (goderbauer) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: I like "supported". 👍

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -13,6 +13,7 @@
 - [ ] I listed at least one issue that this PR fixes in the description above.
 - [ ] I updated/added relevant documentation (doc comments with `///`).
 - [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
+- [ ] I followed the [breaking change policy] and added [Data Driven Fixes] where necessary.
```
Comment (Piinks) [inline .github/PULL_REQUEST_TEMPLATE.md:16]:
```suggestion
- [ ] I followed the [breaking change policy] and added [Data Driven Fixes] where supported.
```

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**